rad/monadic/issue-remote.rad

```diff
@@ -260,6 +276,7 @@
    'add-comment             add-comment
    'edit-comment            edit-comment
    'list-issues             list-issues
+   'get-html                get-html
    'add-admin               (add-admin auth)})
```

Comment:
Should `'set-html` also be an allowed command per default?

Reply:
Fixed